### PR TITLE
Removed test helpers

### PR DESCRIPTION
**Pull request by fmatar**
Signed-off-by: Fady Matar <fmatar@godaddy.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,58 +1,59 @@
...
-    "C*",
-    "cassandra",
-    "ODM",
-    "models",
-    "schemas",
-    "data modeling"
-  ],
-  "author": "GoDaddy Engineers",
-  "license": "MIT"
+	"name": "datastar",
```
Comment (jcrugzz) [inline package.json]: this should still be 2 spaced

File: test/unit/schema.tests.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+var assume            = require('assume'),
+    datastarTestTools = require('datastar-test-tools'),
+    joi               = require('joi-of-cql'),
+    schemas           = require('../fixtures/schemas'),
+    Schema            = require('../../lib/schema');
 
-var debug  = helpers.debug,
-    cql = joi.cql;
+var helpers = datastarTestTools.helpers;
+
```
Comment (jcrugzz) [inline test/unit/schema.tests.js]: remove needless space here and we can just make it part of the same `var` block

Comment (fmatar) [inline test/unit/schema.tests.js]: Usually eslint would complain about this requiring you to split `require` declarations from others.

Comment (jcrugzz) [inline test/unit/schema.tests.js]: it doesnt if its with the var block below it

**Pull request merged**